### PR TITLE
Created Technical Spike Documentation

### PR DESCRIPTION
**Pull request by plasne**
At the suggestion of some of my team members, I have contributed to a section for technical spike documentation. Mostly this includes redacted samples of documentation, but it does also have some suggestions.

### CODE REVIEW

File: design-reviews/recipes/technical-spike.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Technical Spike
+
+From [Wikipedia](<https://en.wikipedia.org/wiki/Spike_(software_development)>)...
+
+A spike in a sprint can be used in a number of ways:
+
+- As a way to familiarize the team with new hardware or software
+- To analyze a problem thoroughly and assist in properly dividing work among separate team members.
+- Spikes tests can also be used to mitigate future risk, and may uncover additional issues that have escaped notice.
```
Comment (TessFerrandez) [inline design-reviews/recipes/technical-spike.md:9]: Spikes tests => spike tests

File: design-reviews/recipes/technical-spike.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+A distinction can be made between technical spikes and functional spikes. The technical spike is used more often for evaluating the impact new technology has on the current implementation. A functional spike is used to determine the interaction with a new feature or implementation.
+
+## Deliverable
+
+Generally the deliverable from a Technical Spike should be a document detailing what was evaluated and the outcome of that evaluation. The specifics contained in the document will vary, but there are some general principals that might be helpful.
+
+In addition, the "technical-spike-samples" folder contains sample documents from previous spikes. "istio-spike.docx" is the most recent. Some information has been redacted.
```
Comment (TessFerrandez) [inline design-reviews/recipes/technical-spike.md:17]: Link to the folder

Comment (TessFerrandez) [inline design-reviews/recipes/technical-spike.md:17]: Please don't add binary files to the repo as they can't be git-diffed... and we also can't PR review them without downloading the branch, nor can we view them in the repo.

If you want to add examples, please convert them to proper markdown files.  
In general we should encourage the use of markdown for documentation that makes it into repos.

**Comment by TessFerrandez** [comment]
Please link the PR to the issue the PR resolves

**Comment by fnocera** [comment]
@plasne thanks so much for the contribution. I think the technical spike recipe is really helpful but would recommend removing the samples as I am not sure they add much in terms of explanation/understanding and are not as broadly applicable as the samples we usually look for.

**Comment by plasne** [comment]
Very well, I will submit for PR just the doc and say "samples available by contacting me"

File: design-reviews/recipes/technical-spike.md (hunk 2 above)
Comment (plasne) [reply design-reviews/recipes/technical-spike.md]: Samples have been removed.

Comment (plasne) [reply design-reviews/recipes/technical-spike.md]: Samples have been removed and a note that they are available on request.

**Comment by charleszipp** [comment]
@TessFerrandez or @fnocera would either of you mind reviewing to see if we can get this merged?

**Review by fnocera — APPROVED** [review]
Looks good, thanks for the contribution!

**Review by TessFerrandez — APPROVED** [review]

**Comment by TessFerrandez** [comment]
@plasne this has been approved for 20 days, it just has some merge conflicts - do you have access to merge or do you want me to?

**Comment by plasne** [comment]
> @plasne this has been approved for 20 days, it just has some merge conflicts - do you have access to merge or do you want me to?

I fixed the merge issue, but I don't seem to have rights to merge this: "Only those with write access to this repository can merge pull requests." If you want to do go ahead and merge and close the PR, that would be great.

**Pull request merged**